x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/AuthenticationService.java

```diff
@@ -554,6 +554,13 @@ private void consumeToken(AuthenticationToken token) {
          */
         // pkg-private for tests
         void handleNullToken() {
+            List<Realm> unlicensedRealms = realms.getUnlicensedRealms();
+            if (unlicensedRealms.isEmpty() == false) {
+                logger.warn("No authentication token can be extracted using realms [{}]." +
```

Comment:
Can we avoid the `token` word? It leads people to look at the token service and ....

In the section below we say `No valid credentials found` which is terminology that could probably be adapted to work here.
```suggestion
                logger.warn("No authentication credential could be extracted using realms [{}]." +
```